aiida_common_workflows/workflows/relax/siesta/workchain.py

```diff
@@ -49,3 +49,4 @@ def convert_outputs(self):
         self.out('stress', res_dict['stress'])
         if 'stot' in self.ctx.workchain.outputs.output_parameters.attributes:
             self.out('total_magnetization', get_magn(self.ctx.workchain.outputs.output_parameters))
+        self.out('remote_folder', self.ctx.workchain.outputs.remote_folder)
```

Comment:
Yes the same change must be done in the other implementations, but I do not know the details of each implementation and therefore I will ask them to do it when they implement their bands part. When all the implementations are updated we can put it as "required". If you do it before, and you merge in master, all the workchains will fail because the required output is not returned. Therefore for the moment I will leave it like that and open an issue later to remind us to change to required.